src/coreclr/nativeaot/Runtime/rhassert.h

```diff
@@ -62,6 +62,6 @@ void Assert(const char * expr, const char * file, unsigned int line_num, const c
 
 #define FAIL_FAST_GENERATE_EXCEPTION_ADDRESS 0x1
 
-#define RhFailFast() RaiseFailFastException(NULL, NULL, FAIL_FAST_GENERATE_EXCEPTION_ADDRESS)
+#define RhFailFast() PalRaiseFailFastException(NULL, NULL, FAIL_FAST_GENERATE_EXCEPTION_ADDRESS)
```

Comment:
```suggestion
#ifdef HOST_WINDOWS
#define RhFailFast() RaiseFailFastException(NULL, NULL, FAIL_FAST_GENERATE_EXCEPTION_ADDRESS)
#else
void RhFailFast();
#endif
```
We should add `RhFailFast` as the PAL API (it needs to be macro to make FAIL_FAST_GENERATE_EXCEPTION_ADDRESS work correctly - see my other comment), and delete  PalRaiseFailFastException/RaiseFailFastException and relate goo from the PAL.